lib/elixir/lib/range.ex

```diff
@@ -21,6 +21,22 @@ defmodule Range do
       iex> last
       3
 
+  A Range includes members from Enum such as reduce, count and member?
```

Comment:
I think we can skip this for `Map` and `List` as it's very natural for them to provide an implementation of the `Enumerable` protocol.